resources/zib/zib-MedicationAdministration2.xml

```diff
@@ -0,0 +1,318 @@
+      </type>
+    </element>
+    <element id="MedicationAdministration.extension:deviatingAdministration">
+      <path value="MedicationAdministration.extension" />
+      <sliceName value="deviatingAdministration" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationAdministration.DeviatingAdministration" />
+      </type>
+    </element>
+    <element id="MedicationAdministration.extension:doubleCheckPerformed">
+      <path value="MedicationAdministration.extension" />
+      <sliceName value="doubleCheckPerformed" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationAdministration.DoubleCheckPerformed" />
+      </type>
+    </element>
+    <element id="MedicationAdministration.extension:medicationAdministrationReasonForDeviation">
+      <path value="MedicationAdministration.extension" />
+      <sliceName value="medicationAdministrationReasonForDeviation" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationAdministration.ReasonForDeviation" />
+      </type>
+    </element>
+    <element id="MedicationAdministration.extension:medicationAdministrationReasonForDeviation.value[x]">
+      <path value="MedicationAdministration.extension.value[x]" />
+      <comment value="This MedicationAdministrationReasonForDeviation extension has overlap with `statusReason`. Both concepts allow the documentation of reasons why an administration did not take place, potentially resulting in duplicated information in both places. This extension, however, has a broader scope because it also covers the reasons why administrations are performed differently than the agreement." />
+    </element>
+    <element id="MedicationAdministration.status">
+      <path value="MedicationAdministration.status" />
+      <short value="MedicationAdministrationStatus" />
+      <definition value="The status of the administration, as a description of the stage within the administering process. Only the status codes ‘completed’ and ‘cancelled’ apply to indivisible products (such as tablets or suppositories). For divisible products (such as infusions), doses can also be ‘suspended’ or ‘aborted’.  &#xD;&#xA;&#xD;&#xA;When documenting this, the following interpretations are used: &#xD;&#xA;&#xD;&#xA;- Active: The product is administered.&#xD;&#xA;- Interrupted: Use has (temporarily) been interrupted, because of a side effect, for example. Later, the patient and/or doctor can decide whether or not to resume or discontinue use.&#xD;&#xA;- Discontinued: Administration has stopped.&#xD;&#xA;- Completed: Administration has been completed.&#xD;&#xA;- Not started: The product was never administered." />
+      <alias value="MedicatieToedieningStatus" />

```

Comment:
cardinality warning missing in the knowni issue

Reply:
Will be added